src/Mvc/benchmarks/Microsoft.AspNetCore.Mvc.Performance/RuntimePerformanceBenchmarkBase.cs

```diff
@@ -49,7 +49,7 @@ private class NullLoggerFactory : ILoggerFactory, ILogger
 
         private class BenchmarkViewExecutor : ViewExecutor
         {
-            public BenchmarkViewExecutor(IOptions<MvcViewOptions> viewOptions, IHttpResponseStreamWriterFactory writerFactory, ICompositeViewEngine viewEngine, ITempDataDictionaryFactory tempDataFactory, DiagnosticListener diagnosticListener, IModelMetadataProvider modelMetadataProvider)
+            public BenchmarkViewExecutor(IOptions<MvcViewOptions> viewOptions, IHttpResponseWriterFactory writerFactory, ICompositeViewEngine viewEngine, ITempDataDictionaryFactory tempDataFactory, DiagnosticListener diagnosticListener, IModelMetadataProvider modelMetadataProvider)
```

Comment:
It would be nice to see some changes that avoid `PagedCharBufferTextWriter` as well as what the impact is on perf 👍